Startup.cs

```diff
@@ -88,5 +175,90 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             }
          });
       }
+
+      private void SeedUsers(IServiceScope serviceScope, ApplicationDbContext applicationDbContext)
```

Comment:
This function in its current state would always add an 'Alice' user in case it does not exist. I would like to see this replaced with a function that adds a default user (maybe also Admin instead of Alice :)?) in case there are no users defined yet.

Reply:
Yea, for the initial setup of the identityService + identity this is the case, but I'll change it :)